blog/2025-03-13.md

```diff
@@ -33,6 +33,34 @@ end-to-end test cases.
 📌[View Full
 Changelog](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/compare/0.3.0...0.4.0)
 
+## Try It Out
+
+With Docker Desktop or Podman installed locally:
+
+```powershell
+git clone https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service
+cd Data-Management-Service
+git checkout 0.4.0
+
+# Start all required services inside of Docker
+cd eng\docker-compose
+cp .env.example .env
+.\start-local-dms.ps1 -EnableSearchEngineUI -EnableConfig -r
+
+# Configure Keycloak
+.\setup-keycloak.ps1
+```
+
+The `.http` files in `src/dms/tests/RestClient` can help you understand how to
+interact with these systems. In particular, see
+`authorization-demonstration-Dec2024.http`. These files should be executable in
+[Visual Studio](https://learn.microsoft.com/en-us/aspnet/core/test/http-files?view=aspnetcore-9.0), VS Code via
+[rest-client](https://marketplace.visualstudio.com/items?itemName=humao.rest-client)
+or
+[httpYac](https://marketplace.visualstudio.com/items?itemName=anweber.vscode-httpyac),
+and
+[Rider](https://www.jetbrains.com/help/rider/Http_client_in__product__code_editor.html).
+
```

Comment:
I wanted to give people a heads up on how to try the release. I tested the commands above with tag `0.4.0`.